jenkins/scripts/integration_test.sh

```diff
@@ -43,6 +43,14 @@ EPHEMERAL_TEST="${EPHEMERAL_TEST:-false}"
 TEST_EXECUTER_VM_NAME="${TEST_EXECUTER_VM_NAME}"
 TEST_EXECUTER_PORT_NAME="${TEST_EXECUTER_PORT_NAME:-${TEST_EXECUTER_VM_NAME}-int-port}"
 TEST_EXECUTER_FIP_TAG="${TEST_EXECUTER_FIP_TAG:-${TEST_EXECUTER_VM_NAME}-floating-ip}"
+IRONIC_INSTALL_TYPE="${IRONIC_INSTALL_TYPE:-rpm}"
+IRONIC_FROM_SOURCE="${IRONIC_FROM_SOURCE:-false}"
+IRONIC_LOCAL_IMAGE_BRANCH="${IRONIC_LOCAL_IMAGE_BRANCH:-}"
+IRONIC_LOCAL_IMAGE="${IRONIC_LOCAL_IMAGE:-}"
```

Comment:
Why do we need these? If the variable is set, it already has the value, otherwise we set it to empty...?